### ISSUE TITLE
"Home" screen

### ISSUE THREAD

**Issue by robertocarroll** [robertocarroll/barjeel-app#3]
The icons don't mean much on their own (at least until the user has used the app a few times): 
![appdesigns1 4-4](https://cloud.githubusercontent.com/assets/1565953/5710396/aa1caa38-9a99-11e4-9ef5-73c021c4f124.jpeg)

The user has to click to get an explanation of what the section means, e.g. 
![appdesigns1 5-5](https://cloud.githubusercontent.com/assets/1565953/5710407/b7ad0076-9a99-11e4-95a7-dc5150a61592.jpeg)

Perhaps the home screen could consolidate the different screens in the current layout into a single screen so that it is clear what each section holds for the user without any further interaction, e.g. extra clicks.

**Comment by UnaJanicijevic** [comment]
I'm sending a pdf with some of these issues addressed.
Not sure we should give a descriptor for each section on the same page—you might agree when you see the mock up. It's gets a bit wordy, and personally I don't mind a bit of mystery, but I agree the icons alone might be a little too vague so maybe an option with icons and names is best, then click for a full page with description.

**Comment by clintmclean** [comment]
Hi, I agree the full text is too much but the icon and title works nicely for me.
For the score vs hearts discussion - I like hearts or lives or whatever too though after you 'die' and use all your hearts maybe it could say  something like Game Over you got 7 out of 10 correct. It then gives kids something to beat so there is a measure of them improving or being better than their peers which makes it more fun I think.

C.

**Comment by robertocarroll** [comment]
Thanks for the update. Closing this issue.
